src/UnitySampleProject/Assets/Tests/Anvil/AnvilProvider.cs

```diff
@@ -0,0 +1,41 @@
+using System.Threading.Tasks;
+using ChainSafe.Gaming.InProcessSigner;
+using ChainSafe.Gaming.Web3;
+using ChainSafe.Gaming.Web3.Environment;
+using ChainSafe.Gaming.Web3.Evm.Wallet;
+using Nethereum.RPC.Accounts;
+using Nethereum.Web3.Accounts;
+
+namespace ChainSafe.Gaming.Unity.Tests
+{
+    /// <summary>
+    /// Anvil account provider.
+    /// </summary>
+    public class AnvilProvider : WalletProvider, IAccountProvider
+    {
+        public IAccount Account { get; private set; }
+
+        public AnvilProvider(Web3Environment environment, IChainConfig chainConfig) : base(environment, chainConfig)
+        {
+            // Initialize account via private key
+            Account = new Account("0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80");
```

Comment:
Is this one of the PRC's default accounts? If so, can we be sure it should always have the same value for everyone in every case?

Reply:
emm as I understood it it's generated from the Mnemonic so if we use the same one we'll be okay